docs/deploy/tensorrt.rst

```diff
@@ -0,0 +1,267 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Relay TensorRT Integration
+==============================================
+**Author**: `Trevor Morris <https://github.com/trevor-m>`_
+
+Introduction
+------------
+
+NVIDIA TensorRT is a library for optimized deep learning inference. This integration will offload as
+many operators as possible from Relay to TensorRT, providing a performance boost on NVIDIA GPUs
+without the need to tune schedules.
```

Comment:
Better to provide an overview of this tutorial, including what information/takeaway readers could expect after reading this article.

After reading through, I would suggest mentioning the following points:
1. (optional) Some model performance numbers to impressive readers.
2. Install TensorRT and build TVM with it.
3. Build a ResNet-18 with TensorRT.
4. Operator support (I also suggest moving this section to the end along with the last section about adding new operators.)
5. Deploy the built model with TensorRT runtime.
6. How to support other operators.